samples/nats_jetstream/v3/sender/main.go

```diff
@@ -0,0 +1,81 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/kelseyhightower/envconfig"
+
+	cejsm "github.com/cloudevents/sdk-go/protocol/nats_jetstream/v3"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+)
+
+const (
+	count = 10
+)
+
+type envConfig struct {
+	// NATSServer URL to connect to the nats server.
+	NATSServer string `envconfig:"NATS_SERVER" default:"http://localhost:4222" required:"true"`
+
+	// Subject is the nats subject to publish cloudevents on.
+	Subject string `envconfig:"SUBJECT" default:"sample" required:"true"`
+}
+
+type Example struct {
+	Sequence int    `json:"id"`
+	Message  string `json:"message"`
+}
+
+func main() {
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		log.Fatalf("Failed to process env var: %s", err)
+	}
+
+	natsURL := "nats://localhost:4222"
+	natsSubject := "sample"
+
+	ctx := context.Background()
+	urlOpt := cejsm.WithURL(natsURL)
+	sender, err := cejsm.NewSender(ctx, natsSubject, urlOpt)
```

Comment:
See comment in the receiver example. Please follow Kafka/HTTP samples where we construct the sender/receiver from the CloudEvents client using protocol instead of lower-level methods.